readme.md

```diff
@@ -144,7 +144,17 @@ queryString.parse('foo=1&foo=2&foo=3');
 Type: `string`\
 Default: `','`
 
-The character used to separate array elements when using `{arrayFormat: 'separator'}`.
+The **single** character used to separate array elements when using `{arrayFormat: 'separator'}`.
```

Comment:
"The character" already means singular. I don't really see the need to point it out.

Reply:
Agree, I'll revert it